### ISSUE TITLE
Java SDK - Single request paramater (instead of pageSize, continuationToken, ...)

### ISSUE THREAD

**Issue by booniepepper** [openfga/sdk-generator#201]

**Comment by jimmyjames** [comment]
I don't think this is an issue any longer? APIs take an options object, e.g., `ClientReadAuthorizationModelsOptions`, `ClientCheckOptions`, etc., which contain various params for that endpoint. @rhamzeh any reason to keep this open?

**Comment by booniepepper** [comment]
Oops, I think I may have just missed closing this issue
